api/api-service.js

```diff
@@ -9,20 +9,19 @@ var Seneca = require('seneca')
 
 var server = new Hapi.Server()
 
-server.connection({ 
+server.connection({
   port: PORT
 })
 
 server.register({
-  register: Chairo, 
+  register: Chairo,
   options:{
     seneca: Seneca({
       tag: 'api',
       log: 'silent',
-      legacy: { logging: false },
-      internal: { logger: require('seneca-demo-logger') },
       debug: {short_logs:true}
     })
+    .use('demo-logger')
```

Comment:
is the log output format now mixed? Does it start with JSON and then move to the demo format? I used `internal: { logger: require('seneca-demo-logger') }` to ensure that logging had the same format from the start